python/pyspark/sql/connect/window.py

```diff
@@ -18,24 +18,23 @@
 
 check_dependencies(__name__)
 
-import sys
 from typing import TYPE_CHECKING, Union, Sequence, List, Optional
 
 from pyspark.sql.column import Column
+from pyspark.sql.window import (
+    Window as ParentWindow,
+    WindowSpec as ParentWindowSpec,
+)
 from pyspark.sql.connect.expressions import (
     ColumnReference,
     Expression,
     SortOrder,
 )
-from pyspark.util import (
-    JVM_LONG_MIN,
-    JVM_LONG_MAX,
-)
 from pyspark.sql.window import Window as PySparkWindow, WindowSpec as PySparkWindowSpec
 from pyspark.errors import PySparkTypeError
 
 if TYPE_CHECKING:
-    from pyspark.sql.connect._typing import ColumnOrName
+    from pyspark.sql.connect._typing import ColumnOrName, ColumnOrName_
```

Comment:
Why do we need `ColumnOrName_`?

Reply:
This change is to make the typing consistent with classic side, which uses `ColumnOrName_`.

The `ColumnOrName_` was introduced in https://github.com/apache/spark/commit/a70006d9a7b578721d152d0f89d1a894de38c25d
 and used in multiple places. Probably we can revisit it and remove it if possible.

Reply:
ah okie. That's fine.